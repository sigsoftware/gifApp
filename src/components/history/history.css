list {
    list-style-type: none;
    color: aliceblue;
}

.text-title {
    text-align: center;
}