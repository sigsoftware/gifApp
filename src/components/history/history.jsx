import React from 'react'
import "./history.css";

let History = (props) => {
    return (
        <>
            <ol className="list">{props.children}</ol>
        </>
    )
}

export { History };
