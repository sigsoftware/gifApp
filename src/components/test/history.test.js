import { shallow } from "enzyme";
import { History } from "../history/history";

describe("Pruebas al componente History", () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<History />);
    })

    test("Probar que la pagina se renderiza correctamente", () => {
        expect(wrapper).toMatchSnapshot();
    })

    test("Probar que existe una lista ordenada en el componente",()=>{
        let ol = wrapper.find("ol");
        let className = ol.prop("className");
        expect(className.includes("list")).toBe(true);
    })

})