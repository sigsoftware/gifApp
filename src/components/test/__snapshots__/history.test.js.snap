// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente History Probar que la pagina se renderiza correctamente 1`] = `
<Fragment>
  <ol
    className="list"
  />
</Fragment>
`;
